test-utils/Cargo.toml

```diff
@@ -6,6 +6,31 @@ edition = "2018"
 license = "Apache-2.0"
 homepage = "https://substrate.dev"
 repository = "https://github.com/paritytech/substrate/"
+autotests = false
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+futures = { version = "0.3.1", features = ["compat"], optional = true }
+substrate-test-utils-derive = { path = "./derive", optional = true }
+tokio = { version = "0.2.13", features = ["macros"], optional = true }
+
+[dev-dependencies]
+sc-service = { path = "../client/service" }
+trybuild = { version = "1.0", features = ["diff"] }
+
+[features]
+macros = [
```

Comment:
Why do we need this?

Reply:
We don't "need" it. I can also remove the feature and the derive will be always included. That's what you want?

Reply:
done (I removed the feature flag)